hidekela/switch_mode.js

```diff
@@ -0,0 +1,20 @@
+var switchThemeBtn = document.querySelector(".switch_btn");
+
+var currTheme = 'light';
+
+function switchTheme() {
+    if(currTheme === 'light')
+    {
+        document.body.setAttribute("class","dark_theme");
+        currTheme = 'dark';
+    }
+    else
+    {
+        document.body.setAttribute("class","light_theme");
+        currTheme = 'light';
+    }
+}
+
```

Comment:
When toggling the `light` and the `dark` theme, as we see here, variables `currTheme` and setter  `document.body.setAttribute` is a constant, so you can use ternary condition to simplify you code like :  

```
function switchTheme() {
        let isLigthTheme = currTheme === 'light';
        document.body.setAttribute("class", isLigthTheme ? 'dark_theme' : 'light_theme');
        currTheme = isLigthTheme ? 'dark' : 'light';
}
```